src/open_inwoner/utils/middleware.py

```diff
@@ -98,6 +98,9 @@ def __call__(self, request):
         if request.path.startswith(self.get_pass_prefixes(request)):
             return self.get_response(request)
 
+        if "REDIRECT_SESSION_VARIABLE" in request.session.__dict__:
```

Comment:
The `request.session` can be indexed like a dict, so you shouldn't use `.__dict__` here
```suggestion
        if request.session.get("REDIRECT_SESSION_VARIABLE"):
```

Also I'm not too sure about the name of `REDIRECT_SESSION_VARIABLE`, in what situation is it expected to be true? And I never see this session variable being removed or set to false, so after becoming true, it will stay that way for that session

Reply:
I ended up with a different (and simpler) solution